### PR TITLE
CANS-1088: Move Domain Level Comments to Domain Footer

### PR DESCRIPTION
**Pull request by DmitriyRudenko**
## Description
https://osi-cwds.atlassian.net/browse/CANS-1088

also small bug fix for read-only assessment: unsaved modal appears when user leaves read-only assessment page when domain was expanded previously

## Tests

- [x] I used TDD to develop the feature
- [x] I have included unit tests
- [x] I have included new acceptance tests or modified existing ones
- [ ] I have included other tests

## Types of changes

- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring (No behavioral changes)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] I ran all my tests locally which were green.
- [x] My code adds no new issues to Code Climate
- [x] I ran all the linters for the project.
- [x] My code is in a stable state ready to be deployable, but not necessarily complete.
- [x] I promise on my honor as a CWDS developer to ensure I don't break the build (or bring donuts if I leave things broken), to check linters, use best practices, to leave the code in better shape than I found it, and to have a good day.

### CODE REVIEW

**Comment by DmitriyRudenko** [comment]
jenkins retest this please

File: app/javascript/Application/components/Assessment/AssessmentContainer.js [hunk 1]
```diff
@@ -216,7 +216,7 @@ export default class AssessmentContainer extends Component {
       assessment,
       assessmentServiceStatus: LoadingState.ready,
       isValidForSubmit,
-      isUnsaved: true,
+      isUnsaved: this.state.isEditable,
```
Comment (LeonidMarushevskyi) [inline app/javascript/Application/components/Assessment/AssessmentContainer.js]: Does this change related to the story?

File: app/javascript/Application/components/Assessment/style.sass [hunk 2]
```diff
@@ -313,7 +313,7 @@ $item-confidential-label-height: 6.05rem
 .domain-comment-accordion-paper
```
Comment (LeonidMarushevskyi) [inline app/javascript/Application/components/Assessment/style.sass]: Should we change this class name to not mention accordion any more?

File: app/javascript/Application/components/Assessment/AssessmentContainer.js (hunk 1 above)
Comment (DmitriyRudenko) [reply app/javascript/Application/components/Assessment/AssessmentContainer.js]: No

Comment (DmitriyRudenko) [reply app/javascript/Application/components/Assessment/AssessmentContainer.js]: See PR description

File: app/javascript/Application/components/Assessment/style.sass (hunk 2 above)
Comment (DmitriyRudenko) [reply app/javascript/Application/components/Assessment/style.sass]: tried to rename it everywhere, it seems I missed this one

Comment (DmitriyRudenko) [reply app/javascript/Application/components/Assessment/style.sass]: done

Comment (LeonidMarushevskyi) [reply app/javascript/Application/components/Assessment/style.sass]: ok

**Pull request merged**